sdk/cloudmachine/Azure.Projects/samples/HelloRAG/Program.cs

```diff
@@ -1,21 +1,22 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Azure.AI.OpenAI;
 using Azure.Projects;
 using Azure.Projects.OpenAI;
 using OpenAI.Chat;
 using OpenAI.Embeddings;
 
 ProjectInfrastructure infrastructure = new();
-infrastructure.AddFeature(new OpenAIModelFeature("gpt-35-turbo", "0125"));
-infrastructure.AddFeature(new OpenAIModelFeature("text-embedding-ada-002", "2", AIModelKind.Embedding));
+infrastructure.AddFeature(new OpenAIChatFeature("gpt-35-turbo", "0125"));
```

Comment:
I think the 35-turbo model is deprecated. We may want to change it.

Reply:
Yeah, I was actually trying to change it, but failed because of various quotas, redetections, etc. I will try one more time next week.